services/ocm/README.md

```diff
@@ -13,40 +13,110 @@ Internal GRPC APIs:
 * ocmcore: used for creating federated shares on the receiver side
 * authprovider: authenticates webdav requests using the ocm share tokens
 
+## Enable OCM
+
+To enable OpenCloudMesh you have to set three environment variables:
```

Comment:
```suggestion
To enable OpenCloudMesh you have to set three environment variables. The path  `/etc/ocis` in the example below depends on the installation type and derives, if not otherwise defined from the `OCIS_CONFIG_DIR` envvar.
```
Make it clear where the path comes from

Reply:
Please rebase, there have been some small changes in the readme that should not get overwritten...

Reply:
As discussed with @butonic I take care